pllug-presentation-system/pandoc-workflow/pandocrunner.cpp

```diff
@@ -0,0 +1,64 @@
+#include "pandocrunner.h"
+
+#include <QCoreApplication>
+#include <QDir>
+
+#include <QDebug>
+
+PandocRunner::PandocRunner(const QStringList &params, QObject* parent):
+    QProcess(parent)
+  , mParams(params)
```

Comment:
I think, it would be better idea to hide all parameters in class realization. We can use enum class for document types, for example:
``` cpp
enum class PandocFormat {
   HTML,
   Markdown,
   MarkdownGithub,
   Json
 //...
}
```
Than user can pass only parameter he needed to method:
``` cpp
QString PandocRunner::run(PandocFormat from, PandocFormat to) 
{
   //...
}
```
What do you think?